x-pack/plugins/cases/public/components/create/flyout/use_cases_add_to_new_case_flyout.test.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* eslint-disable react/display-name */
+
+import { renderHook } from '@testing-library/react-hooks';
+import React from 'react';
+import { CasesContext } from '../../cases_context';
+import { CasesContextStoreActionsList } from '../../cases_context/cases_context_reducer';
+import { useCasesAddToNewCaseFlyout } from './use_cases_add_to_new_case_flyout';
+
+describe('use cases add to new case flyout hook', () => {
+  const dispatch = jest.fn();
+  let wrapper: React.FC;
+  beforeEach(() => {
+    dispatch.mockReset();
+    wrapper = ({ children }) => {
+      return (
+        <CasesContext.Provider
```

Comment:
Is there any particular reason where the `TestProvider` component cannot be used?

Reply:
I think that the `TestProvider` accepts props. You can pass a dispatch function if you like. It can be optional and defaulted to a noop function.

Reply:
the test is working and passing as it is. Is there any particular reason you'd like me to use the `TestProvider`?